.github/workflows/test_s3_minio.yml

```diff
@@ -4,9 +4,6 @@ name: S3/Minio Exploratory Test
 
 on:
   push:
-    branches:
-      - main
-      - real_world_s3_tests
```

Comment:
do we want this to run on every push (including feature branch pushes)? I don't think it's needed that frequently